### ISSUE TITLE
Get video duration in the specified format

### ISSUE THREAD

**Issue by artyuum** [PHP-FFMpeg/PHP-FFMpeg#549]
Hi, is there a way to specify a format for the returned duration time ?
Example for a video of 5 seconds, the following code returns the following value : 
```php
$duration = $ffprobe->format('file.mp4')->get('duration');
echo $duration;

// Output
5
```
I'd like to be able to add the [sexagesimal](https://trac.ffmpeg.org/wiki/FFprobeTips#Formatcontainerduration) option in order to get the duration time in the following format :  `HOURS:MM:SS.MICROSECONDS`.

Is it possible to do that with the current version of your wrapper ?

**Comment by jens1o** [comment]
No, it's not. But it should be possible with little code of yourself.https://github.com/PHP-FFMpeg/PHP-FFMpeg/blob/0b871e59e7a809033fa27c3086588e72c662e9ab/src/FFMpeg/Coordinate/TimeCode.php#L67-L91

But I'd be willing to add another method to `TimeCode` to make it accessible for anyone. :)
